cities_light/migrations/0009_auto_20171229_1656.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.8 on 2017-12-29 15:56
+from __future__ import unicode_literals
+
+import autoslug.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cities_light', '0008_city_timezone'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Department',
```

Comment:
I like that it's the same word as in my language but should this not be SubRegion ?